lib/flappi/definition.rb

```diff
@@ -343,6 +343,15 @@ def request_example(*v)
       set_example('request', v)
     end
 
+    # Define an API error
+    def api_error(status_code, field_name, field_description)
+      endpoint_info[:api_errors] << {
+        status_code: status_code,
+        response_field_name: field_name,
+        response_field_description: field_description
+      }
+    end
+
```

Comment:
We don't differentiate here between errors that return a JSON body and errors that return a text reason?